packages/perseus/src/widgets/label-image/marker.tsx

```diff
@@ -17,23 +17,19 @@ import {iconCheck, iconChevronDown, iconMinus} from "../../icon-paths";
 import {AnswerPill} from "./answer-pill";
 
 import type {IconType} from "../../components/icon";
-import type {
-    AnalyticsEventHandlerFn,
-    InteractiveMarkerType,
-} from "@khanacademy/perseus-core";
 import type {CSSProperties} from "aphrodite";
 
-type Props = InteractiveMarkerType & {
+type Props = {
+    selected?: string[];
+    showCorrectness?: "correct" | "incorrect";
+    label: string;
```

Comment:
Left out `answers`, `x`, and `y`. Focus is already included down below. X and y don't seem to be used, but if they are added back as Props, there's no error. Since they don't seem to be used by Marker though, I just left them out.